Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -1,16 +1,65 @@
 #!/usr/bin/env python3
 
+import subprocess
 import json
+import time
 from az import *
+from helios import *
 import os
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 f = open('../cohesity.json',)
 data = json.load(f)
 resource_group = data['resource_group']
 workspace_name = data['workspace_name']
-playbook_name = data['playbook_name']
 apiKey = data['apiKey']
 f.close()
 
 assert has_dup_incidents(resource_group, workspace_name) == False
```

Comment:
Please move this [down](https://github.com/cohesity/Azure-Sentinel/pull/8/commits/0181cc6743413b2fc1a3ef975b8b38ed4a8f0331#diff-962835dddd48b7def1a7bed8c5b7a8ec6391a111f42c8f1da2cc8f13bbc70cb7R62),  so that  it's clubbed with the testing functions. Infact, please move all the above code to the end of the file so that there's clear seperation between functions and when we're calling.

Reply:
done